Index: src/test/java/event/ReminderApp/service/EventServiceTest.java
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
--- src/test/java/event/ReminderApp/service/EventServiceTest.java	(date 1595235610382)
+++ src/test/java/event/ReminderApp/service/EventServiceTest.java	(date 1595235610382)
@@ -0,0 +1,28 @@
+package event.ReminderApp.service;
+
+import org.junit.jupiter.api.Test;
+
+import static org.junit.jupiter.api.Assertions.*;
+
+class EventServiceTest {
+
+    @Test
+    void addEvent() {
+    }
+
+    @Test
+    void getEventById() {
+    }
+
+    @Test
+    void getAllEvents() {
+    }
+
+    @Test
+    void updateEvent() {
+    }
+
+    @Test
+    void deleteEvent() {
+    }
+}
\ No newline at end of file
Index: src/test/java/event/ReminderApp/dao/EventDatabaseServiceTest.java
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
--- src/test/java/event/ReminderApp/dao/EventDatabaseServiceTest.java	(date 1595235610357)
+++ src/test/java/event/ReminderApp/dao/EventDatabaseServiceTest.java	(date 1595235610357)
@@ -0,0 +1,28 @@
+package event.ReminderApp.dao;
+
+import org.junit.jupiter.api.Test;
+
+import static org.junit.jupiter.api.Assertions.*;
+
+class EventDatabaseServiceTest {
+
+    @Test
+    void insertEvent() {
+    }
+
+    @Test
+    void selectAllEvents() {
+    }
+
+    @Test
+    void selectEventById() {
+    }
+
+    @Test
+    void updateEventById() {
+    }
+
+    @Test
+    void deleteEventById() {
+    }
+}
\ No newline at end of file
Index: src/test/java/event/ReminderApp/ReminderAppApplicationTest.java
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
--- src/test/java/event/ReminderApp/ReminderAppApplicationTest.java	(date 1595235610512)
+++ src/test/java/event/ReminderApp/ReminderAppApplicationTest.java	(date 1595235610512)
@@ -0,0 +1,12 @@
+package event.ReminderApp;
+
+import org.junit.jupiter.api.Test;
+
+import static org.junit.jupiter.api.Assertions.*;
+
+class ReminderAppApplicationTest {
+
+    @Test
+    void main() {
+    }
+}
\ No newline at end of file
Index: src/test/java/event/ReminderApp/datasource/PostgresDataSourceTest.java
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
--- src/test/java/event/ReminderApp/datasource/PostgresDataSourceTest.java	(date 1595235610432)
+++ src/test/java/event/ReminderApp/datasource/PostgresDataSourceTest.java	(date 1595235610432)
@@ -0,0 +1,12 @@
+package event.ReminderApp.datasource;
+
+import org.junit.jupiter.api.Test;
+
+import static org.junit.jupiter.api.Assertions.*;
+
+class PostgresDataSourceTest {
+
+    @Test
+    void hikariDataSource() {
+    }
+}
\ No newline at end of file
Index: src/test/java/event/ReminderApp/api/EventControllerTest.java
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
--- src/test/java/event/ReminderApp/api/EventControllerTest.java	(date 1595235610407)
+++ src/test/java/event/ReminderApp/api/EventControllerTest.java	(date 1595235610407)
@@ -0,0 +1,28 @@
+package event.ReminderApp.api;
+
+import org.junit.jupiter.api.Test;
+
+import static org.junit.jupiter.api.Assertions.*;
+
+class EventControllerTest {
+
+    @Test
+    void addEvent() {
+    }
+
+    @Test
+    void getAllEvents() {
+    }
+
+    @Test
+    void getEventById() {
+    }
+
+    @Test
+    void updateEvent() {
+    }
+
+    @Test
+    void deleteEvent() {
+    }
+}
\ No newline at end of file
Index: src/main/resources/application.yml
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.BaseRevisionTextPatchEP
<+>app:\r\n  datasource:\r\n    jdbc-url: jdbc:postgresql://localhost:5432/reminderapp\r\n    username: postgres\r\n    password: postgres\r\n    pool-size: 30\r\n
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
--- src/main/resources/application.yml	(revision 8fbed0ed9581063ddf97b14e7a1444402ec6409b)
+++ src/main/resources/application.yml	(date 1595235610473)
@@ -2,5 +2,5 @@
   datasource:
     jdbc-url: jdbc:postgresql://localhost:5432/reminderapp
     username: postgres
-    password: postgres
+    password: Orisicare123
     pool-size: 30
Index: src/test/java/event/ReminderApp/model/EventTest.java
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
--- src/test/java/event/ReminderApp/model/EventTest.java	(date 1595235610540)
+++ src/test/java/event/ReminderApp/model/EventTest.java	(date 1595235610540)
@@ -0,0 +1,28 @@
+package event.ReminderApp.model;
+
+import org.junit.jupiter.api.Test;
+
+import static org.junit.jupiter.api.Assertions.*;
+
+class EventTest {
+
+    @Test
+    void getId() {
+    }
+
+    @Test
+    void getName() {
+    }
+
+    @Test
+    void getStartDate() {
+    }
+
+    @Test
+    void getEndDate() {
+    }
+
+    @Test
+    void getDetails() {
+    }
+}
\ No newline at end of file
